draft-ZIP-0226.rst

```diff
@@ -0,0 +1,303 @@
+    - if `split = 1` set :math:`\psi' = \mathsf{randomSample}`, otherwise set :math:`\psi' = \psi^{old}`
+
+**Enabling Backwards Compatibility with ZEC Notes:** the following constraints must be added to enable backwards compatibility with the Orchard ZEC notes.
+
+The old note commitment is computed using a “rolling-aggregate” sinsemilla commitment. This means that the commitment is computed by adding new chunks or windows to the accumulated value. This method will be used in order to maintain a single commitment instance for the old note commitment, that will be used both for Orchard ZEC notes and for ZSA notes. The original Orchard ZEC notes will be conserved and not actually be converted into ZSA notes, as we will always need to compute them.
+
+- The input note in the old note commitment integrity must either include a type (ZSA note) or not (ZEC-Orchard note)
+    - If the type auxiliary input is set :math:`\mathsf{type}_\mathbb{P}` = :math:`\mathcal{V}^\mathsf{Orchard}`
+        - NoteCommitment has a “compatibility” path that computes the note commitment as in plain Orchard (i.e.: without including the type)
+        - This path also uses the original domain separator for ZEC note commitment
+    - Else, 
+        - The NoteCommitment adds the type, :math:`\mathsf{type}_\mathbb{P}`, as a final “chunk” of the Sinsemilla commitment
+        - The NoteCommitment uses a different domain separator for ZSA note commitment
+
+
+Backward Compatibility
+----------------------
+
+In order to have a "clean" backwards compatibility with the ZEC notes, we have designed the circuit to support both ZEC and ZSA notes. As we specify above, there are three main reasons we can do this:
+- The input notes with a type denote the ZSA custom assets, generating a note commitment that includes the type; whereas the notes without a type, denote the ZEC notes, and generate a note commitment that does not include the type, in order to maintain the referencability to the Merkle tree
+- The value commitment is abstracted to allow for the value base-point as a variable private input to the proof
+- The ZEC-based actions will still include dummy input notes, whereas the ZSA-based actions will include split input notes
+
+Burn Mechanism
+==============
+The burn mechanism may be needed for off-boarding the wrapped assets from the chain, or enabling advanced tokenomics on native tokens. It is part of the Issuance/Burn protocol, but given that it can be seen as an extension of the Transfer protocol, we add it here for readability.
+
+In essence, the burn mechanism is a transparent / revealing extension to the transfer protocol that enables a specific amount of any asset type to be sent into “oblivion”. Our burn mechanism does NOT send assets to a non-spendable address, it simply reduces the total number of assets in circulation at the consensus level. It is enforced at the consensus level, by using an extension of the value balance mechanism used for ZEC assets.
```

Comment:
```suggestion
In essence, the burn mechanism is a transparent / revealing extension to the transfer protocol that enables a specific amount of any Asset type to be sent into “oblivion”. Our burn mechanism does NOT send Assets to a non-spendable address, it simply reduces the total number of units of a given Asset in circulation at the consensus level. It is enforced at the consensus level, by using an extension of the value balance mechanism used for ZEC Assets.
```

Reply:
Done in [PR#649](https://github.com/zcash/zips/pull/649).